src/controllers/CommandController/utils.js

```diff
@@ -143,7 +143,7 @@ export default class CommandUtils {
       )
       response.msg = `Olá ${
         user.name.split(' ')[0]
-      }! Atualmente tu estás no nível ${user.level}.
+        }! Atualmente tu estás no nível ${user.level.value}.
```

Comment:
Nao to conseguindo sugerir na linha correta, mas na linha 155 tem que mudar pra:

`text: `${user.score.value} pontos de reputação no ranking geral!``